build/Dockerfile

```diff
@@ -6,36 +6,38 @@ RUN dpkg --add-architecture i386 \
 	&& apt-get update -y \
 	&& apt-get install -y --no-install-recommends ca-certificates curl lib32gcc1 lib32stdc++6 libcurl4-gnutls-dev:i386 \
 	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-# Install gosu.
```

Comment:
Are we going to change the `gosu` more thant the `apt-get` part? I don't think so. I think that if we are going to change something it'll be add or remove some `apt-get`. I don't see any cache optimization there. Any way, if you prefer to split it is ok for me. It's not a big deal.